usr/share/rear/restore/NBKDC/default/90_create_missing_directories.sh

```diff
@@ -0,0 +1,9 @@
+#
+# some backup SW doesn't restore mountpoints properly
+#
+#
+# create missing directories
+pushd /mnt/local >&8
+mkdir -p mnt proc sys tmp dev/pts dev/shm
```

Comment:
Is this not already covered by `restore/default/90_create_missing_directories.sh`? Did you try without and it did not work? IMHO this file can be removed.